client/src/components/add-job-lead-component/index.jsx

```diff
@@ -160,8 +155,7 @@ function AddJobLead({
                   <Button onClick={handleClose}>CANCEL</Button>
                   <Button
                     onClick={() => {
-                      handleClose();
-                      handleResetInputs();
+                      navigate("/dashboard");
```

Comment:
i think it's better to navigate back to job leads dsahboard not the home dashboard